src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -637,6 +637,11 @@ static bool IsRelevantDocument(SyntaxTreeIndex syntaxTreeInfo)
             void CollectMatchingReferences(ISymbol originalUnreducedSymbolDefinition, SyntaxNode node,
                 ISyntaxFactsService syntaxFacts, ISemanticFactsService semanticFacts, ArrayBuilder<FinderLocation> locations)
             {
+                if (syntaxFacts.IsImplicitObjectCreation(node))
```

Comment:
@jcouv In 

https://github.com/dotnet/roslyn/pull/43645/files#diff-aeace626a93e2448b1aec4bc466c9931R336-R337

I see you introduced an extension method called "IsImplicitObjectCreationExpression". So I'm not sure whether I should use `IsImplicitObjectCreation` instance method, or `IsImplicitObjectCreationExpression`, and why we need the two?